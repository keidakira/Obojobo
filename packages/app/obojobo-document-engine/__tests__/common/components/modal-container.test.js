import React from 'react'
import renderer from 'react-test-renderer'

import ModalContainer from '../../../src/scripts/common/components/modal-container'

describe('ModalContainer', () => {
	test('ModalContainer component', () => {
		const component = renderer.create(<ModalContainer />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
