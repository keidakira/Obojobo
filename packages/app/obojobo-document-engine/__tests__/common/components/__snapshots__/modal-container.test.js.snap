// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalContainer ModalContainer component 1`] = `
<div
  className="obojobo-draft--components--modal-container"
>
  <div
    className="content"
  />
</div>
`;
